priv/migrations/01_create_published_messages.exs

```diff
@@ -0,0 +1,17 @@
+defmodule :"#{Application.get_env(:railway_ipc, :repo)}.Migrations.CreatePublishedMessages" do
+  use Ecto.Migration
+
+  def change do
+    create table(:published_messages, primary_key: false) do
```

Comment:
What do you think about database constraints on these fields being required?

Reply:
yea let's do it.